openapi-spec/swagger.yaml

```diff
@@ -831,6 +899,44 @@ paths:
         required: true
         schema:
           type: string
+      - name: limit
+        in: query
+        description: Number of query results on multiple pages
+        schema:
+          minimum: 1
+          type: integer
+          format: int32
+      - name: offset
+        in: query
+        description: Start position of the pagination query
+        schema:
+          minimum: 0
+          type: integer
+          format: int32
+      - name: sortKey
+        in: query
+        description: Sorting keyword, which can be sorted based on the number of initiators
+        schema:
+          type: string
+          enum:
+          - size
+          - status
+          - availabilityZone
+      - name: sortDir
+        in: query
+        description: Asending or descending
+        schema:
+          type: string
+          enum:
+          - asc
+          - des
+      - name: name
+        in: query
```

Comment:
Did you consider other query filters. The spec doesn't include all the query parameters for the getting the required response based on the filters.
Please add the supported query filters also